config/settings/production.py

```diff
@@ -157,6 +157,11 @@
         # Runs on Tuesday
         "schedule": crontab(day_of_week=2, hour="6", minute="0"),
     },
+    "every-week-remove-old-report-data": {
+        "task": "adserver.tasks.remove_old_report_data",
+        # Runs on Wednesday
```

Comment:
I wonder if it's faster to just run it nightly, so there's never more than 1 day of data to delete? I don't think it'll make a huge difference, but perhaps this comment could note why we're doing it weekly instead of nightly, if there's a specific reason?

Reply:
There's no real reason. I am not sure how fast it'll be but I assume it's going to be slow the first time and reasonably fast after that.